### PR TITLE
pyupgrade --py37-plus **/*.py

### PR DESCRIPTION
**Pull request by cclauss**
[`pyupgrade --py37-plus **/*.py`](https://github.com/asottile/pyupgrade/blob/main/README.md)

Python f-strings are shorter, easier to read, and faster.  https://www.scivision.dev/python-f-string-speed

### CODE REVIEW

File: pytype/tools/merge_pyi/test_data/imports_multiple.pep484.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """Test multiple from-imports."""
 from typing import List, Text
 
-def f() -> List[Text]:
+def f() -> List[str]:
```
Comment (rchen152) [inline pytype/tools/merge_pyi/test_data/imports_multiple.pep484.py:4]: Could you revert this change? This is a test file anyway, and it doesn't really make sense for Text to be imported but not used.

**Review by rchen152 — CHANGES_REQUESTED** [review]
Thanks for this PR! Just one minor comment.

File: pytype/tools/merge_pyi/test_data/imports_multiple.pep484.py (hunk 1 above)
Comment (cclauss) [inline pytype/tools/merge_pyi/test_data/imports_multiple.pep484.py:4]:
```suggestion
def f() -> List[Text]:
```

**Review by rchen152 — APPROVED** [review]
Thanks! I'll merge this once the checks finish running.
